Add item OpenCV in User Guide (#2796)
openvinotoolkit/cvat: cvat/apps/documentation/user_guide.md

  - [Track mode (advanced)](#track-mode-advanced)
  - [Attribute annotation mode (advanced)](#attribute-annotation-mode-advanced)
  - [AI Tools](#ai-tools)

>   - [OpenCV](#opencv)

Reviewer: ```suggestion
  - [OpenCV](#opencv-tools)
```

- This action will automatically annotates one frame.
  In the [Automatic annotation](#automatic-annotation) section you can read how to make automatic annotation of all frames.

> ## OpenCV

Reviewer: ```suggestion
## OpenCV tools
```

> This tools based on Open CV, it is used to create polygons. To work with tool, you need to download it.

Reviewer: The tool based on [Open CV](https://opencv.org/) Computer Vision library which is an open-source product that includes many CV algorithms. Some of these algorithms can be used to simplify the annotation process.

Reviewer: First step to work with OpenCV is to load it into CVAT. Click on the toolbar icon, then click `Load OpenCV`.

> The tool based on [Open CV](https://opencv.org/) Computer Vision library which is an open-source product that includes many CV algorithms. Some of these algorithms can be used to simplify the annotation process.

Reviewer 2: OpenCV should be written together

To load OpenCV, click on the toolbar icon, then click `Load OpenCV`.

  ![](static/documentation/images/image198.jpg)

Once it is loaded, the tool's functionality will be available.

> ### Drawing

Reviewer: ```suggestion
### Intelligent scissors 
```

> Drawing is an AI method of creating a polygon by placing points with automatic drawing of a line between them.

Reviewer: ```suggestion
Intelligent scissors is a CV method of creating a polygon by placing points with automatic drawing of a line between them.
```

> ### intelligent scissors

Reviewer: ```suggestion
### Intelligent scissors
```
